composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "php-translation/loco-adapter",
+    "name": "etsglobal/loco-adapter",
```

Comment:
please keep the original package name so that we can propose a PR from this fork to the original repo (in order not to maintain our fork if this PR is accepted and merged in the original repo)

Reply:
I think @lucianduta tried that, but he couldn't get the package to load, because composer will try to load the open-source `php-translations/loco-adapter` from Github, instead of loading our fork.

Reply:
It should work though, since it's what we're doing for other forks, for example for doctrine migration bundle: https://github.com/ETSGlobal/BUSINESS/blob/master/composer.json#L99 & https://github.com/ETSGlobal/BUSINESS/blob/master/composer.json#L18

Reply:
Hm, maybe it only works when using a tag that does not exist in the base repo.

@lucianduta Here's a full tutorial https://medium.com/swlh/using-your-own-forks-with-composer-699358db05d9 that should work

Reply:
https://github.com/ETSGlobal/loco-adapter/pull/2